packages/paste-core/components/tooltip/src/Tooltip.tsx

```diff
@@ -60,42 +62,111 @@ export interface TooltipProps extends TooltipPrimitiveInitialState {
-  ({ baseId, children, element = "TOOLTIP", state, text, ...props }, ref) => {
+const Tooltip = React.forwardRef<HTMLDivElement, TooltipVariantProps>(
+  ({ baseId, children, element = "TOOLTIP", state, text, actionHeader, keyCombinationsActions, ...props }, ref) => {
     const tooltip = state || useTooltipPrimitiveState({ baseId: `paste-tooltip-${useUID()}`, ...props });
+
     return (
       <>
         {React.Children.only(
           <TooltipPrimitiveReference {...tooltip} ref={ref} {...children.props}>
             {(referenceProps) => React.cloneElement(children, referenceProps)}
-          </TooltipPrimitiveReference>,
+          </TooltipPrimitiveReference>
         )}
         <TooltipPrimitive element={element} {...tooltip} {...props} as={StyledTooltip}>
           {/* import Paste Theme Based Styles due to portal positioning. */}
           <StyledBase>
             <TooltipArrow {...tooltip} />
-            <Text
-              element={`${element}_TEXT`}
-              as="span"
-              color="colorTextInverse"
-              fontSize="fontSize20"
-              lineHeight="lineHeight10"
-            >
-              {text}
-            </Text>
+            {text && !keyCombinationsActions && (
+              <Text
+                element={`${element}_TEXT`}
+                as="span"
+                color="colorTextInverse"
+                fontSize="fontSize20"
+                lineHeight="lineHeight10"
+              >
+                {text}
+              </Text>
+            )}
+            {keyCombinationsActions && (
+              <Stack orientation="vertical" spacing="space30">
+                {actionHeader && (
+                  <Text element={`${element}_HEADER`} fontWeight="fontWeightSemibold" as="p" color="colorTextInverse">
+                    {actionHeader}
+                  </Text>
+                )}
+                {keyCombinationsActions.map((action, idx) => (
+                  <Box display="flex" key={`action-${idx}`} justifyContent="space-between">
+                    {action.name && (
+                      <Text
+                        element={`${element}_ACTION_TEXT`}
+                        as="span"
+                        color="colorTextInverse"
+                        fontSize="fontSize20"
+                        lineHeight="lineHeight10"
+                        marginRight="space70"
+                      >
+                        {action.name}
+                      </Text>
+                    )}
+                    <KeyboardKeyGroup
+                      variant="inverse"
+                      element={`${element}_ACTION_KEY_GROUP`}
+                      disabled={action.disabled}
+                    >
+                      {action.eventKeyCombination.map((key, i) => (
+                        <KeyboardKey key={`key-${idx}-${i}`} element={`${element}_ACTION_KEY`}>
+                          {key}
+                        </KeyboardKey>
```

Comment:
todo: The text in Keyboard Keys should have the same font size and line height as "Mac"/"Windows"

Reply:
Done!